controller/release.go

```diff
@@ -71,12 +72,33 @@ func (r *ReleaseRepo) Add(data interface{}) error {
 
 	err = r.db.QueryRow("INSERT INTO releases (release_id, artifact_id, data) VALUES ($1, $2, $3) RETURNING created_at",
 		release.ID, artifactID, data).Scan(&release.CreatedAt)
+	if err != nil {
+		return err
+	}
 
 	release.ID = postgres.CleanUUID(release.ID)
 	if release.ArtifactID != "" {
 		release.ArtifactID = postgres.CleanUUID(release.ArtifactID)
 	}
-	return err
+
+	r.createEvent(release)
+
+	return nil
+}
+
+func (r *ReleaseRepo) createEvent(release *ct.Release) {
+	logErr := func(err error) {
+		log.Printf("Error creating release event: %s\n", err)
+	}
+	data, err := json.Marshal(release)
+	if err != nil {
+		logErr(err)
+		return
+	}
+	query := "INSERT INTO events (object_id, object_type, data) VALUES ($1, $2, $3)"
+	if err := r.db.Exec(query, release.ID, string(ct.EventTypeRelease), data); err != nil {
+		logErr(err)
+	}
```

Comment:
The event creation methods are all pretty similar, could refactor to a more generic:

```go
func createEvent(objectID string, objectType ct.EventType, v interface{})
```